pkg/utils/cel/quantity.go

```diff
@@ -47,7 +47,7 @@ type Quantity struct {
 // NewQuantity creates a new Quantity
 func NewQuantity(q *resource.Quantity) Quantity {
 	if q == nil {
-		q = resource.NewScaledQuantity(0, resource.Nano)
+		q = resource.NewMilliQuantity(0, resource.DecimalSI)
```

Comment:
Use `NewQuantity` instead `NewMilliQuantity`, the Milli is still too small

Reply:
> [!note]
> https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
> Kubernetes doesn't allow you to specify CPU resources with a precision finer than 1m or 0.001 CPU. To avoid accidentally using an invalid CPU quantity, it's useful to specify CPU units using the milliCPU form instead of the decimal form when using less than 1 CPU unit.

Milli is required because CPU resources are measured in millicores (m) as the basic unit.